### PR TITLE
Rm content related to Im-Online Pallet

### LINKED ISSUE #5800: Being offline is no longer a slashable offense
https://wiki.polkadot.network/docs/learn-staking-advanced#unresponsiveness

_For every session, validators will send an "I'm online" heartbeat to indicate they are live. If a validator produces no blocks during an epoch and fails to send the heartbeat, it will be reported as unresponsive. Slashing may occur depending on the repeated offenses and how many other validators were unresponsive or offline during the epoch._

This changes with runtime 1.2.0

_Yes, there's not chilling anymore for being offline since it's no longer a slashable offence. The only punishment is getting less rewards (since offline validators don't participate in consensus/sign statements nor author blocks)._

PR:  https://github.com/polkadot-fellows/runtimes/pull/178

### PR DESCRIPTION
**Pull request by filippoweb3**
- Unresponsiveness
- im-online session keys

### CODE REVIEW

**Comment by filippoweb3** [comment]
@Overkillus, can you please check if I removed the right content? Essentially, I removed im-online pallet-related stuff (slashing for unresponsiveness, heartbeat content, and online messages).

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -487,15 +486,12 @@ particular levels are not implemented or referred to in the code or in the syste
...
 calculated, see the equations in the section below.
 
-- Level 1: isolated [unresponsiveness](./learn-staking-advanced.md/#unresponsiveness), i.e. being
-  offline for an entire session. Generally no slashing, only [chilling](#chilling).
-- Level 2: concurrent unresponsiveness or isolated
-  [equivocation](./learn-staking-advanced.md/#equivocation), slashes a very small amount of the
-  stake and chills.
-- Level 3: misconducts unlikely to be accidental, but which do not harm the network's security to
+- Level 1: Isolated [equivocation](./learn-staking-advanced.md/#equivocation), slashes a very small
+  amount of the stake and chills.
```
Comment (Overkillus) [inline docs/learn/learn-staking.md]: It will no longer chill you.

File: docs/maintain/maintain-guides-how-to-chill.md [hunk 2]
```diff
@@ -64,12 +64,10 @@ different.
...
-If a validator was unresponsive for an entire session, the validator bond would be chilled in a
-process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may
-restrict the validator from being selected in the next election depending on the session in which it
-was chilled (see considerations above). A chilled validator may re-declare the intent to validate at
-any time. However, it is recommended that the validator attempts to determine the source of the
-chill before doing so.
+Involuntary chilling restricts a validator from being selected in the next election depending on the
+session in which it was chilled (see considerations above). A chilled validator may re-declare the
+intent to validate at any time. However, it is recommended that the validator attempts to determine
+the source of the chill before doing so.
```
Comment (Overkillus) [inline docs/maintain/maintain-guides-how-to-chill.md]: Validators will no longer be chilled for offences. Prior to recent changes chilling was caused by ImOnline, Invalid Backing or Equivocations. ImOnline was removed altogether and involuntary chilling was completely disabled with the merge of https://github.com/paritytech/polkadot-sdk/pull/2226 (soon to be released).

Check this issue for a more comprehensive list of changes: https://github.com/orgs/paritytech/projects/119/views/15?pane=issue&itemId=61684472

**Review by Overkillus — COMMENTED** [review]
Added some relatively important nits but overall looking nice. Check here for a more comprehensive list of changes: https://github.com/orgs/paritytech/projects/119/views/15?pane=issue&itemId=61684472

I will also do one more scan of the whole staking section of the wiki to check if we didn't miss anything.

**Comment by Overkillus** [comment]
Did a quick scan and found some stale mentions:
- `:::warning Single Operators with Multiple Validators` as it mentions ImOnline within Learn -> Nominator
- `The removal of a percentage of an account's DOT as a punishment for a validator acting maliciously
or incompetently (e.g., equivocating or remaining offline for an extended period).` in Glossary -> Slashing
- `#### Commission Fees & Slashes
The network slashes a validator for a misbehavior (e.g. validator offline, equivocation, etc.) ` -> mentions offline slashes
- `and equivocation offenses are slashed at much higher rates than equivalent offline
offenses.`
- `Slashing will happen if a validator misbehaves (e.g. goes offline, attacks the network, or runs
modified software)`
- `Validators perform critical functions for the network, and as such, have strict uptime requirements. [...]` -> mentions getting ImOnline slashed a bit further
- `Remember, even if your validator goes offline for some time, the offline slash is much more
forgiving than the equivocation or parachain validity slashing.`
- `Slashing penalties for being offline are much less than those for equivocation.`

Please re-check if you missed some others if possible. I searched for `offline`, `online` keywords but hopefully those are all the mentions.

Not connected to disabling but I spotted it:
- `The minimal safe ratio of validators per parachain is 5:1. With a sufficiently large set of
validators, the randomness of their distribution along with` -> we currently use 3:1 in Kusama and we believe it is safe.

File: docs/learn/learn-staking.md (hunk 1 above)
Comment (filippoweb3) [reply docs/learn/learn-staking.md]: removed

File: docs/maintain/maintain-guides-how-to-chill.md (hunk 2 above)
Comment (filippoweb3) [reply docs/maintain/maintain-guides-how-to-chill.md]: rm involuntary chill section

**Review by DrW3RK — APPROVED** [review]
Looks good to me.

**Pull request merged**